src/SLCore/Service/Rules/Models/EffectiveRuleDetailsDto.cs

```diff
@@ -36,8 +34,7 @@ public record EffectiveRuleDetailsDto(
     VulnerabilityProbability? vulnerabilityProbability,
     [property: JsonConverter(typeof(EitherJsonConverter<RuleMonolithicDescriptionDto, RuleSplitDescriptionDto>))]
     Either<RuleMonolithicDescriptionDto, RuleSplitDescriptionDto> description,
-    List<EffectiveRuleParamDto> @params);
+    [JsonProperty("params")] List<EffectiveRuleParamDto> parameters) : IRuleDetails;
```

Comment:
not sure if this looks better

Reply:
It is needed for the implemenation of the IRuleDetails.
The EffectiveIssueDetailsDto and EffectiveRuleDetailsDto are very similar, which is why this interface was introduced. 
This also prevents code duplication of converting 99% similar Dtos to RuleInfo in the SLCoreRuleMetaDataProvider